packages/@lightningjs/ui-components/src/docs/LightningResources.mdx

```diff
@@ -0,0 +1,39 @@
+﻿{/* prettier-ignore */}
+{/*
+Copyright 2023 Comcast Cable Communications Management, LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+\*/}
+
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Docs/Lightning Resources" />
+
+## Additional Resources
```

Comment:
Just being picky, I think all of the header types can be bumped up by one here. So instead of this having two hashes, since it's the title of the page, make it one, and same for everything below (remove one hash)